lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -42,7 +42,7 @@ const defaultItemsCountFormatter = (kind: string) => (n: number) => `${n} item${
     defaultAvailableItemsCountFormatter = defaultItemsCountFormatter('available'),
     defaultSelectedItemsCountFormatter = defaultItemsCountFormatter('transferred');
 
-export default function NxTransferList<T extends string | number>(props: Props<T>) {
+function NxTransferList<T extends string | number>(props: Props<T>) {
```

Comment:
No need for this change, you can still export here and then assign the propTypes to it below.  See `NxPagination` for example.

Reply:
oo thanks didn't know that; this is fixed in the following commit:
8df61c7ba4c2ad10415a869de9993da4bfe5b9af